src/pages/FeatureFlagMissing.tsx

```diff
@@ -0,0 +1,29 @@
+import { EmptyState, Text, TextLink } from '@grafana/ui';
+import React from 'react';
+
+export default function FeatureFlagMissing() {
+  return (
+    <EmptyState variant="call-to-action" message="Missing feature flag.">
+      <p>
+        The Grafana Advisor requires the <code>grafanaAdvisor</code> feature toggle to be enabled.
+      </p>
+      <p>
+        <TextLink
+          href="https://grafana.com/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/"
+          external
+        >
+          Instructions to enable the feature toggle:
```

Comment:
Nit: maybe?
```suggestion
          Instructions to enable the feature toggle in the grafana.ini:
```

Reply:
well, it depends on the environment, it's not always modifying the grafana.ini. For example for Grafana cloud, the documentation says to open a ticket with support 🤷

Reply:
Aaa, ok 👍